client/v3/client_test.go

```diff
@@ -30,7 +30,9 @@ import (
 )
 
 func NewClient(t *testing.T, cfg Config) (*Client, error) {
-	cfg.Logger = zaptest.NewLogger(t)
+	if cfg.Logger == nil {
+		cfg.Logger = zaptest.NewLogger(t)
```

Comment:
Why do we want a named client in the `tests/integration/testing.go` but not here?

Reply:
Fixed.

Reply:
Thanks!